Packs/Uptycs/pack_metadata.json

```
@@ -5,12 +5,13 @@
     "currentVersion": "1.0.1",
```

pack version should be the same as the latest rn file.
```suggestion
    "currentVersion": "1.0.2",
```